weather_mv/loader_pipeline/pipeline.py

```diff
@@ -145,6 +145,11 @@ def run(argv: t.List[str]) -> t.Tuple[argparse.Namespace, t.List[str]]:
     if known_args.zarr_kwargs and not known_args.zarr:
         raise ValueError('`--zarr_kwargs` argument is only allowed with valid Zarr input URI.')
 
+    if known_args.zarr_kwargs:
+        if not known_args.zarr_kwargs.get('start_date') or not known_args.zarr_kwargs.get('end_date'):
+            logger.warning('`--zarr_kwargs` not contains both `start_date` and `end_date`'
```

Comment:
Let's not use a logger.warning. We want to do something like:
```
import warnings
warnings.warn("This is a warning")

```

Reply:
Code Updated.